web/pdf_find_bar.js

```diff
@@ -189,6 +189,7 @@ class PDFFindBar {
 
   toggle() {
     if (this.opened) {
+      PDFFindController.reset();  // To reset the Input Bar when Search Button closed
```

Comment:
Let's handle this in the `close()` method instead since it should also happen if the dialog is closed in other ways than via `toggle()`.

Reply:
okay. I'll try that.